### DISCUSSION TITLE
WPF的ListView如何实现列的显示与隐藏

### DISCUSSION THREAD

**Discussion by CrazyZhang666** [BYJRK/DotNet-Discussions#10] [WPF 开发]
想实现任务管理器这种可以自定义显示列的效果，这个怎么实现呢？

![image](https://user-images.githubusercontent.com/28080853/213123113-58bd90fe-a1c4-4853-b31d-cfd1409a485d.png)

**Comment by BYJRK** [comment]
你可以手动写 `DataGrid.Columns`，然后给每个 `Column` 的 `Visibility` 属性绑定到一个 `CheckBox`（或者 `MenuItem`）的 `IsChecked` 属性（当然，你还需要一个经典的 `BoolToVisibilityConverter` 去做转换
